back/boxtribute_server/business_logic/statistics/crud.py

```diff
@@ -429,6 +432,9 @@ def compute_stock_overview(base_id):
             boxes.c.box_state,
             Product.category.alias("category_id"),
             fn.TRIM(fn.LOWER(Product.name)).alias("product_name"),
+            fn.CONCAT(
+                fn.ROUND(boxes.c.measure_value * Unit.conversion_factor, 2), Unit.symbol
```

Comment:
please round and group by absolute measure value and not relative to the display unit.